src/include/mallocMC/creationPolicies/Scatter.hpp

```diff
@@ -946,9 +946,9 @@ namespace mallocMC
                                                 "or AlignmentPolicy.");
                 }
                 auto initKernel
-                    = [] ALPAKA_FN_ACC(const AlpakaAcc& acc, T_DeviceAllocator* heap, void* heapmem, size_t memsize) {
-                          heap->pool = heapmem;
-                          heap->initDeviceFunction(acc, heapmem, memsize);
+                    = [] ALPAKA_FN_ACC(const AlpakaAcc& m_acc, T_DeviceAllocator* m_heap, void* m_heapmem, size_t m_memsize) {
+                          m_heap->pool = m_heapmem;
+                          m_heap->initDeviceFunction(m_acc, m_heapmem, m_memsize);
```

Comment:
Why is this rename necesary? and the `m_` convention seems wrong for function parameters.

Reply:
Yes the compiler is showing: `error: declaration of
‘size_t memsize’ shadows a parameter [-Werror=shadow]`